kolibri/core/assets/src/views/connection-snackbars/index.vue

```diff
@@ -49,22 +53,51 @@
     },
     computed: {
       disconnected() {
-        return this.currentSnackbar === ConnectionSnackbars.DISCONNECTED;
+        return !this.connected && this.currentSnackbar === ConnectionSnackbars.DISCONNECTED;
```

Comment:
Is there a case where it would connected and the snackbar would still be set to DISCONNECTED?

Reply:
I added this because I was getting some weird state inconsistencies (because the two mutations are dispatched seperately).